omp/matrix/dense_kernels.cpp

```diff
@@ -368,6 +344,8 @@ void convert_to_hybrid(std::shared_ptr<const OmpExecutor> exec,
     }
 
     size_type coo_idx = 0;
+    // This parallelization may cause the COO part to not being sorted by row
```

Comment:
Maybe add a `NOTE:` or `TODO:` prefix so that this can be found by a grep

Reply:
or `FIXME` actually

Reply:
Good idea, will do that.